faq/ProoferFAQ.php

```diff
@@ -13,6 +12,10 @@
 <h1>Beginning Proofreaders' <br> Frequently Asked Questions (FAQ)</h1>
 <h3 align="right">Version 1.7, released May 27, 2004 </h3>
 
+<hr>
+<p class='warning'>The content for this FAQ document has been moved into the wiki and is no longer maintained in the code. If you see this message, please notify a site admin.</p>
```

Comment:
Maybe put the email or mechanism to contact site admin.

Reply:
Done.